salt/kubernetes-common/init.sls

```diff
@@ -0,0 +1,9 @@
+/etc/kubernetes/config:
```

Comment:
This will conflict with [my kubernetes-common](https://github.com/kubic-project/salt/pull/186/files#diff-d1fc4951f00f2000a4e57d72366b959eR1).